docs/src/chapters/podsumowanie-realizacji.adoc

```diff
@@ -1,3 +1,71 @@
+funkcjonalności sieci Bitcoin oraz Lightning. Stworzony produkt jest kompletnym rozwiązaniem wykorzystującym
+młodą jeszcze technologię, której potencjał nie został jeszcze w pełni wykorzystany. Sprawia to, że projekt ma
+szansę zyskać uznanie wśród społeczności technologi Blockchain.
+
+// TODO tej fajny wykres z jiry
+//.Wykres obrazujący łączny nakład prac w projekcie.
+//image::../images/cumulative_report.png[]
+
+W trakcie prac nad implementacją projektu zrealizowano łącznie 156 zadań. Ich estymacja w postaci punktów historii
+wyniosła 392. Zespół nie śledził, ile czasu zostało poświęcone na zadania. Można przyjąć, że 1 punkt warty był 4
+godziny. Daje to wtedy wynik 1568 godzin poświęconych na realizację implementacji projektu. Jest to jednak wynik
+bardzo nieprecyzyjny i należy go interpretować tylko jako ciekawostkę.
+
+.Wykres obrazujący ilość punktów historii poświęcony na każdy z elementów projektu.
+image::../images/total_storypoints.png[]
+
+.Wykres pokazujący ilość zadań dotyczących poszczególnych elementów projektu.
+image::../images/total_tasks.png[]
+
+Najwięcej nakładów pracy wymagał serwer aplikacji. Zespół ukończył 49 zadań wartych 124 punkty historii. Na
+drugim miejscu uplasowała się dokumentacja, z 38 zadaniami wartymi 104 punty. Ta kategoria uwzględnia zarówno
+pisanie pracy dyplomowej, jak i poprawki do pozostałych dokumentów oraz stworzenie i zarządzanie projektem _asciidoc_.
+Następną kategorią jest aplikacja internetowa. Zrealizowano również 38 zadań, tym razem wartych 93 punty historii.
+Najmniej zadań dotyczyło aplikacji mobilnej. Tej kategorii dotyczyło 17 zdań, których estymacja wynosi 52 punkty.
+Ukończono również wiele mniejszych zadań, niepasujących do wymienionych wcześniej kategorii. Są to na przykład
+poprawki w repozytorium projektu, pisanie testów E2E oraz tworzenie plików _readme_. Tych zadań było jedynie 14,
+a ich wartość to 19 punktów. W trakcie realizacji projektu nie zrealizowano wszystkich zadań. W rejestrze znajduje się
+dalej 10 zadań, większość z nich nie ma jeszcze ustalonej estymacji. Ich realizacja planowana jest w następnych
+wydaniach.
+
+.Wkład członków zespołu w realizację projektu na podstawie zrealizowanych punktów historii.
+image::../images/storypoints_by_person.png[]
+
+Największy udział w implementacji projektu miał Aleksander Konieczny. Zrealizował on zadania warte 262 punkty historii.
+Jest to aż 65.2% wszystkich dostarczonych punktów. Jan Pieczul dostarczył 120 punktów historii, dających 29,9%.
+Sebastianowi Lewandowskiemu udało się ukończyć zadania o wartości 20 punktów historii, co daje 5% udziału w
+implementacji. Statystyki te nie uwzględniają pierwszego etapu historii. W tamtym czasie nad nauką, dokumentacją i
+projektowaniem systemu pracował cały zespół. Autorem makiety interaktywnej był w całości Jan Pieczul, natomiast przy
+prototypie rozkład udziału jest podobny jak w przypadku implementacji końcowego produktu.
+
+// TODO contributions z githuba
```

Comment:
komentarze

Reply:
to samo co wyżej, celowy zabieg żeby nie robić nowego co 5 minut